batch/src/main/java/org/openmrs/analytics/FhirEtl.java

```diff
@@ -130,7 +139,7 @@ static OpenmrsUtil createOpenmrsUtil(String sourceUrl, String sourceUser, String
 	        throws CannotProvideCoderException {
 		FhirSearchUtil fhirSearchUtil = createFhirSearchUtil(options, fhirContext);
 		// TODO remove fhirContext from fhirStoreUtil.
-		FhirStoreUtil fhirStoreUtil = new FhirStoreUtil(options.getGcpFhirStore(), fhirContext);
+		FhirStoreUtil fhirStoreUtil = new FhirStoreUtil(options.getFhirStoreUrl(), fhirContext.getRestfulClientFactory());
```

Comment:
I don't think we need `fhirStoreUtil` in this function anymore, please remove it; ditto `parquetUtil` (this is of course an issue from before this PR).